Docs review, changes (#569)
configcat/docs: website/docs/requests.mdx

#### Hosted

Runs on dedicated infrastructure provided by ConfigCat.

> The basic package includes:

Reviewer: Is this infrastructure setup still relevant? Haven't we migrated the hosted plans to AWS? Or will we very soon?

Author: We will migrate them to AWS, but this plan and setup is still relevant.

#### On-Premise (Self-hosted)

Runs on the customer's own infrastructure. We suggest <a href="https://configcat.com/support/" target="_blank">contacting ConfigCat's engineering</a>

> team for exact requirements and performance details.

Reviewer: Might be a slightly better word choice:

```suggestion
team for exact requirements and performance characteristics.
```

## config JSON downloads

The ConfigCat SDK, which you import into your applications, downloads your feature flags and settings in the
form of a config JSON file from the ConfigCat CDN and caches it locally.

## `GetValue()` call is NOT a config JSON download

Reading feature flag and setting values from cache with `GetValue()` is not considered as a config JSON download.
If the cache is empty (e.g., upon application launch) or expired, a config JSON will be downloaded, and all 
subsequent `GetValue()` calls will then be served from the cache.

## Example use cases

### Frontend/mobile/desktop applications

> Typically, you have a high number of frontend/mobile/desktop application instances. Your user count determines their number.

Reviewer: ```suggestion
Typically, you have a high number of frontend/mobile/desktop application instances. Your user count determines this number.
```

> **15,000** _(users)_ × **300** _(config JSON downloads/user/month)_ = **4,500,000 config JSON downloads / month**

#### Example: A mobile application running on 5k devices 24/7

Having a mobile app that runs on the devices as a background process. The ConfigCat SDK runs in Auto polling mode with 1 hour polling interval.
Your application runs on 5,000 devices.

> **5,000** _(devices)_ × **730** _(hours in a month)_ = **3,650,000 config JSON downloads / month**

### Backend applications

Backend applications typically have fewer instances than frontend applications.

#### Example: Average frequency polling in 4 instances

Let's say you have an API for your frontend application, and you have 4 instances of it behind a load balancer.
All these 4 instances use the ConfigCat SDK in Auto Polling mode with a 1-minute polling interval.

> **4** _(instances)_ × **43,800** _(minutes in a month)_ = **175,200 config JSON downloads / month**

#### Example: High frequency polling in 10 instances

Suppose you want your system to react faster after changing a feature flag in ConfigCat. You can decrease

> the default polling interval down up to 1 second. In this case, we are calculating with 10 running instances.

Reviewer: ```suggestion
the default polling interval all the way down to 1 second. In this case, we are calculating with 10 running instances.
```

### Use webhooks

In a backend application, you may want to consider using [Webhooks.](./advanced/notifications-webhooks.mdx) This way your application receives notifications about changes and downloads the `config JSON` only when needed.

### Call your backend instead of the ConfigCat CDN

In the case of a frontend application, you can reduce the number of calls made towards the ConfigCat CDN by moving the evaluation logic from the frontend application to your backend.

### Use the ConfigCat Proxy to cache/proxy config JSON downloads

The [ConfigCat Proxy](/advanced/proxy/proxy-overview/) allows you to host a feature flag evaluation and config JSON proxy/cache service within your own infrastructure.  

> The [CDN proxy](/advanced/proxy/endpoints/#cdn-proxy) feature allows you to proxy/cache config JSON downloads, and the ConfigCat SDKs can be routed to the ConfigCat Proxy running in your own infrastructure, greatly reducing the number of config JSON downloads.

Reviewer: Relative URLs?

### Purchase additional quota

> You can purchase an additional quota for your subscription to increase your limits. [Contact us](https://configcat.com/support/) for more information.

Reviewer: Same issue with the word "quota" as above.

Author: changed it
